services/director-v2/src/simcore_service_director_v2/modules/dask_client.py

```diff
@@ -372,7 +376,7 @@ def _comp_sidecar_fct(
                         done_dask_callback,
                         task_to_future_map=self._taskid_to_future_map,
                         user_callback=callback,
-                        main_loop=asyncio.get_event_loop(),
+                        main_loop=asyncio.get_event_loop(),  # TODO: SAN how would changing e.g. to uvloop affect this?
```

Comment:
you can revert that file too.
my PR #2798 will change thee things. and this is gone.